<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>601</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>회원가입</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>90</y>
     <width>110</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>아이디</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>240</y>
     <width>110</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>전화번호</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>190</y>
     <width>110</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>이름</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>140</y>
     <width>110</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>비밀번호</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>290</y>
     <width>110</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>가구원수</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_id">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>90</y>
     <width>210</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_pw">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>140</y>
     <width>210</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_name">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>190</y>
     <width>210</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_phone">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>240</y>
     <width>210</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_fam">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>290</y>
     <width>210</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_join_commit">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>360</y>
     <width>180</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>회원가입</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
